scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -24,6 +24,13 @@ workflow ImportGenomes {
 
   String docker_final = select_first([docker, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
 
+  call SetLoadLock {
+    input:
+      output_directory = output_directory,
+      preemptible_tries = preemptible_tries,
+      docker = docker_final
```

Comment:
I don't think we need to pass in a docker image for this. It is just doing google commands (for this task ahaessly@gmail.comd ReleaseLoadLock). I use     
docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:305.0.0"

Reply:
great point! double-checked that that docker has the uuid generator (it does), and have now changed to the cloud-sdk docker.